### PR TITLE
Text cleaning

### PR DESCRIPTION
**Pull request by elicox0**
New extraction function for Texas; uses BeautifulSoup and doesn't show HTML font definitions when the bill text is unavailable. Also removes the large chunks of whitespace for readability.

Deleted the extraction function for Texas; all functionality was added to the sanitize module. "Clean" function from extract.utils was removed and incorporated into sanitize.py as well. Initializes a list of sanitizers before looping through bill version objects so that cleaning can be done with pre-compiled regular expressions (faster).

### CODE REVIEW

File: extract/common.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import re
 import tempfile
 import textract
+from bs4 import BeautifulSoup as bs
```
Comment (MoHayat) [inline extract/common.py]: do we still need bs4 for the extractors?

File: sanitize.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    """Removes line numbers from extracted text."""
+    def __init__(self):
+        self.re = re.compile(r"^ *line *\d+", flags=re.MULTILINE) 
+
+class NewlineCleaner(Sanitizer):
+    """Removes excessive newlines from extracted text."""
+    def __init__(self):
+        self.re = re.compile(r"\n(?=\n)")
+
+class TX_CSS_Cleaner(Sanitizer):
```
Comment (MoHayat) [inline sanitize.py]: super trivial, but snake case and all caps are usually for constants, maybe something like `TexasCSSCleaner` is better here

**Comment by MoHayat** [comment]
@elicox0 was just able to review this PR, and it's looking really good. No changes from my end with the exception of a few convention/lib things.

A couple of notes re: the CA extractors, for clarification:
1. the sanitizer here assumes we will be using the PDFs from CA which is why we're filtering out the `line \d+` right? 
2. if so, would the sanitizer that adds structure to the raw HTML text be in a separate PR? we mentioned this last time we spoke, but let me know if you wanted me to clarify this.

awesome work overall 🙌

File: extract/common.py (hunk 1 above)
Comment (elicox0) [reply extract/common.py]: No, I just missed this line when I was cleaning up. Next push I'll have it removed.

**Comment by elicox0** [comment]
@MoHayat  I made the changes you pointed out, reflected in the most recent commit. re structuring the raw text: yes, I am planning to include that in a separate PR. The current sanitizer is assuming that some of the raw text from CA might include line numbers (I don't know whether that means PDFs or not, but if those only show up in the PDFs then yes). Let me know if you'd like me to make any changes to that or if we won't actually be using any PDFs; this is meant to be structured so that if we decide to remove that sanitizer, all it takes is deleting it from the dictionary under the "ca" heading in sanitize.py.

**Pull request merged**